testing_api/rest.go

```diff
@@ -0,0 +1,97 @@
+package rest
```

Comment:
what is the reason for the name testing_api?

Reply:
Updated to New directory structure ,
```
|_ kubera-api-testing/
    |__kubera
      |__ connection/
      |    |__ connection.go
      |__ authentication/
           |__ authentication_test.go
           |__ authentication.go
```